redash/models/__init__.py

```diff
@@ -63,6 +63,56 @@ def get(self, query_id):
 scheduled_queries_executions = ScheduledQueriesExecutions()
 
 
+@python_2_unicode_compatible
+class TableMetadata(db.Model):
```

Comment:
Should this maybe use the `TimestampMixin` to be able to track when the metadata was created and updated?